### PR TITLE
Use the __main__.py special filename for main.py

### PR DESCRIPTION
**Pull request by Steffo99**
This allows users to run the program directly with `py -m tg` instead of `py -m tg.main`.

### CODE REVIEW

**Comment by lightme16** [comment]
@Steffo99 Hello, thanks for PR.
To be consistent you may want to update https://github.com/paul-nameless/tg/blob/master/pyproject.toml#L16 and README  as well.
But, why don't you like to use `python tg/main.py` cmd? I can't see any  problems with current approach right now, so this PR doesn't look like noticeable improvement.

File: readme.md [hunk 1]
```diff
@@ -46,7 +47,8 @@ From sources:
 pip3 install python-telegram
 git clone git@github.com:paul-nameless/tg.git
 cd tg
-PYTHONPATH=. python3 tg/main.py
+PYTHONPATH=. 
+python3 tg/main.py
```
Comment (lightme16) [inline readme.md:51]: so, need command should be `python  -m tg`, isn't it? then `PYTHONPATH` could be omitted.

**Comment by Steffo99** [comment]
(Sorry if I didn't answer yet, I'm a bit busy at the moment... I should've marked this pull request as draft sooner.)

File: readme.md (hunk 1 above)
Comment (Steffo99) [reply readme.md:51]: Now that I think of it... You're using `flit` as a package manager, so the readme should reflect that...

**Comment by Steffo99** [comment]
> But, why don't you like to use python tg/main.py cmd? I can't see any problems with current approach right now, so this PR doesn't look like noticeable improvement.

Well, there are no problems with it by itself, but using __main__ is a more pythonic way to do it and allows for [some extra functionality] :)

File: readme.md [hunk 2]
```diff
@@ -31,35 +31,43 @@ TODO:
...
 ```
 
-Arch Linux users can install from the AUR: https://aur.archlinux.org/packages/telegram-tg-git/
+### From the AUR
+
+If you're using Arch Linux, you can install tg through [its AUR package](https://aur.archlinux.org/packages/telegram-tg-git/):
 
 ```bash
-yay -S telegram-tg-git
+pacman -S telegram-tg-git
```
Comment (lightme16) [inline readme.md:70]: that's wrong command, it will work only for packages in the official repo.

File: readme.md [hunk 3]
```diff
@@ -31,35 +31,43 @@ TODO:
 
 ## Installation
 
-`python3.8` required.
```
Comment (lightme16) [inline readme.md:34]: why did you delete this? it is worth to mention that python 3.8 is required. e.g it won't work with python3.7 and lower

File: readme.md (hunk 1 above)
Comment (lightme16) [reply readme.md]: agree, it is worth to add this to README. it will make life easier for people who will be adding brew package and others

File: readme.md [hunk 4]
```diff
@@ -31,33 +31,43 @@ TODO:
...
+> Install it with:
+> ```sh 
+> pip3 install flit
+> ```
 
 ```sh
-pip3 install python-telegram
 git clone git@github.com:paul-nameless/tg.git
 cd tg
-PYTHONPATH=. python3 tg/main.py
```
Comment (lightme16) [inline readme.md:49]: i think it is still worth to mention about ` pip3 install python-telegram & python3 -m tg` method. because you don't required to install `flit` to use tg client

File: readme.md (hunk 2 above)
Comment (Steffo99) [reply readme.md:70]: Oh, right, it's in the AUR. Oops. My bad, I'll revert this immediately.

File: readme.md (hunk 3 above)
Comment (Steffo99) [reply readme.md:34]: All installation methods currently listed in the readme won't allow you to install tg with an unsupported Python version...

It wasn't my intention to remove that, though... I forgot to add it back while changing the headers, sorry :disappointed:

File: readme.md (hunk 4 above)
Comment (Steffo99) [reply readme.md:49]: I'm not sure that mixing package managers would be a great idea, but I can add that anyways.

I was thinking of recommending the creation of a virtualenv, is that ok for you?

File: readme.md [hunk 5]
```diff
@@ -28,54 +28,87 @@ TODO:
...
+> Ensure you have the correct version of Python installed before using this method!
+
 ```sh
-docker run -it --rm tg
+git clone https://github.com:paul-nameless/tg.git
+cd tg
+python3 -m venv venv
+source venv/bin/activate
+pip install python-telegram
+python3 tg/__main__.py
```
Comment (lightme16) [inline readme.md:92]: why aren't you using `python3 -m tg` here? I think idea of this PR was so allow using such way of running a module :slightly_smiling_face:

File: readme.md [hunk 6]
```diff
@@ -28,54 +28,87 @@ TODO:
...
-git clone git@github.com:paul-nameless/tg.git
+git clone https://github.com:paul-nameless/tg.git
 cd tg
-PYTHONPATH=. python3 tg/main.py
+flit install
+tg
 ```
 
-Docker (voice recordings and notifications won't work):
+### Without installing
```
Comment (lightme16) [inline readme.md:82]: `Without installing` -> `Using virtualenv` would be better naming

File: readme.md (hunk 5 above)
Comment (Steffo99) [reply readme.md:92]: Well, it isn't installed as a module, so using `-m` won't work.

However, now that I think of it, this should work:
```suggestion
python3 tg
```

File: readme.md (hunk 6 above)
Comment (Steffo99) [reply readme.md:82]: How about:
```suggestion
### Running as a script
```
?

You don't really install the tg package with that method.

Comment (lightme16) [reply readme.md:82]: So, so let use `Running with virtualenv` then?

**Comment by Steffo99** [comment]
Done! I suggest squashing this commit, as it includes many useless subcommits...
